_posts/2019-06-26-cockpit-197.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Cockpit 197
+author: kkoukiou
+date: 2019-06-26
+tags: cockpit linux
+slug: cockpit-197
+category: release
+summary: Machines new features
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly. Here are the release notes from version 197.
+
+### Machines: Support disks that reside on all pool types when adding disks to VMs
```

Comment:
This heading is long.  Something shorter like "support all storage pool types" or something is better, with the full explanation in the text below.